### ISSUE TITLE
Question about avoiding certain tokens

### ISSUE THREAD

**Issue by iamgroot42** [collinzrj/adversarial_decoding_legacy#1]
Thanks for providing the code to your work- it is very well structured and easy to understand! I just had a small question regarding one of the steps for processing logits for the causal_llm_tokenizer:

https://github.com/collinzrj/adversarial_decoding/blob/11535a3d8b1a8138f454c258169ed45cded1bed7/adversarial_decoding.py#L161-L162

For context, the token IDs 628 and 198 correspond to tokens for ` pl` and `\n`. I understand why we would want to avoid the latter, but what is the reason for avoiding 628?

**Comment by collinzrj** [comment]
Hi, I think this might be a bug, commenting out this should also work
